agent-ini/src/IniParser.cc

```diff
@@ -534,15 +534,15 @@ int IniParser::parse()
 	    }
 	    else
 	    {
-		if ((*ff).second.changed ())
+		if ((*tgt_fi).second.changed ())
 		{
-		    if (scanner_start (*f))
-			y2error ("Cannot open %s.", *f);
+		    if (scanner_start (*tgt_f))
+			y2error ("Cannot open %s.", *tgt_f);
 		    else
 		    {
-			y2debug ("File %s changed. Reloading.", *f);
-			FileDescr fdsc (agent.targetPath(*f).c_str());
-			multi_files [*f] = fdsc;
+			y2debug ("File %s changed. Reloading.", *tgt_f);
+			FileDescr fdsc (agent.targetPath(*tgt_f).c_str());
```

Comment:
Now that the names are clear, it turns out we are calling `targetPath` once too many here. Our diligence has paid off!